library/spit_results.py

```diff
@@ -328,6 +328,43 @@ def process_id_u_jboss(fact_names, host_vars):
             'Error: unexpected output from "id -u jboss": %s' % raw_output}
 
 
+JBOSS_EAP_COMMON_DIRECTORIES = 'jboss.eap.common-directories'
+
+
+def process_jboss_common_dirs(fact_names, host_vars):
```

Comment:
maybe better to call out eap in the function name?
`process_jboss_eap_common_dirs`

Also do you think we should start extracting these into a utilities file instead of putting them all in spit_results.py?

Reply:
Function name changed!

Reply:
I absolutely want to pull these out of spit_results.py, but I remember there is something special you have to do to get Ansible to ship multiple files together for a module. I'd like to handle the refactor by moving the current `spit_results` functionality into the rho executable and out of an Ansible role first. That will also have other benefits, like letting it share utility functions with the rest of rho (currently I have had to duplicate some stuff because Ansible modules have their own search path) and letting spit_results log to the terminal, which would *really* help with debugging changes. Would you like me to file an issue for that?

Reply:
@noahl I like your proposal